### ISSUE TITLE
(CS0246: The type or namespace name 'DuckableAttribute' could not be found

### ISSUE THREAD

**Issue by denis-bredun** [byme8/DuckInterface#2]
Hi! I downloaded your nuget-packet to test duck typing. I wrote the same code as you did in README, but it doesn't work.

![(CS0246: The type or namespace name 'DuckableAttribute' could not be found (are you missing 8
using directive or an assembly reference?)](https://github.com/byme8/DuckInterface/assets/48459415/1f151d3b-236c-42ed-a70d-4bf83f704546)
I use .NET 8.0, C# 11.
I wonder if you have any idea why it doesn't work?

Thanks in advance!

**Comment by byme8** [comment]
The documentation is not up to date. In the latest version that deployed to NuGet, there is no need for `` [DuckableAttribute] ``. You can apply the extension method `` Duck<T> `` to any interface.

``` csharp
using DuckInterface;

namespace expDuckable
{
    public class Program
    {
        public static void Main(string[] args)
        {
            var calculator = new AddCalculator();
            var result = ApplyCalculator(calculator.Duck<ICalculator>(), 1, 2);

            Console.WriteLine($"Result: {result}");
        }

        static float ApplyCalculator(ICalculator calculator, float a, float b)
        {
            return calculator.Calculate(a, b);
        }
    }

    public interface ICalculator
    {
        float Calculate(float a, float b);
    }

    public class AddCalculator
    {
        public float Calculate(float a, float b)
        {
            return a + b;
        }
    }
}
```

**Comment by byme8** [comment]
If you want to try the old way with the `` Duckable `` attribute. It is available in the 0.1.5 version of the NuGet.

**Comment by friendly-neighborr** [comment]
@byme8 , Oh, I see, thank you very much!
